pkg/logs/input/docker/launcher.go

```diff
@@ -263,6 +267,7 @@ func (l *Launcher) restartTailer(containerID string) {
 		source = oldTailer.source
 		if l.collectAllSource != nil {
 			l.collectAllSource.RemoveInput(containerID)
+			l.collectAllSource.RemoveInfo(containerID)
```

Comment:
Should we do this in `RemoveInput` ?

Reply:
We could. However if we decide to use this input dictionary for other types of sources - the key could be different or how we manage it could change based on on the scenario. (Input's are not used in call tailing scenarios for example)

Reply:
OK, got it :+1: